dfusion_rust_core/src/models/solution.rs

```diff
@@ -13,16 +13,15 @@ pub struct Solution {
 }
 
 impl Solution {
-    pub fn trivial(num_orders: usize) -> Self {
+    pub fn trivial(num_orders: usize, num_tokens: usize) -> Self {
```

Comment:
Shouldn't num_tokens inside the trivial solution be always 0?

Reply:
At first, this is what I was hoping we could do and I would very much like it to be this way, however, this would require adapting solution evaluation for trivial solutions since we get missing token errors on trivial solutions with no price vector. 

Here:

https://github.com/gnosis/dex-services/blob/2a442f847dfe716b83a5784528910a0ad6abb60d/driver/src/snapp.rs#L40-L63


I was hoping you would point this out and encourage me to solve this problem in a better way. So I will go ahead and do that. It would require less overall changes anyway.


note that the above code fails with `TokenNotFoundError` on the following trivial solution

```
Solution {
    prices: []
    executed_buy_amounts: [0, 0], 
    executed_sell_amounts: [0, 0],
}
```